### PR TITLE
display a dialog when not connected to the network

### LINKED ISSUE #314: Short term handling for crash on startup with no networking
The longer term solution will be #185 

This is meant for a short term exception handler and to throw an alert

### PR DESCRIPTION
**Pull request by sashei**
Fixes #314 

## To Do

- ~~add “WIP” to the PR title if pushing up but not complete nor ready for review~~
- [x] double check the original issue to confirm it is fully satisfied
- [x] request the "UX" team perform a design review (if/when applicable)
- [x] make sure CI builds are passing (e.g.: fix lint and other errors)

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -80,6 +80,9 @@
     <string name="missing_url">MISSING_URL</string>
     <string name="default_avatar_url">https://firefoxusercontent.com/00000000000000000000000000000000</string>
     <string name="syncing_entries">Syncing your entries…</string>
+    <string name="OK">OK</string>
```
Comment (linuxwolf) [inline app/src/main/res/values/strings.xml]: nit: string names ought to match in case mapping (lower case ASCII)

**Comment by linuxwolf** [comment]
I see the following on my physical device when:

* Previously signed into FxA
* device is in airplane mode
* starting up Lockbox

```
2018-12-21 11:48:11.510 6939-6939/mozilla.lockbox W/mozilla.lockbo: Accessing hidden method Landroid/view/View;->computeFitSystemWindows(Landroid/graphics/Rect;Landroid/graphics/Rect;)Z (light greylist, reflection)
2018-12-21 11:48:11.511 6939-6939/mozilla.lockbox W/mozilla.lockbo: Accessing hidden method Landroid/view/ViewGroup;->makeOptionalFitsSystemWindows()V (light greylist, reflection)
2018-12-21 11:48:11.592 6939-6973/mozilla.lockbox W/mozilla.lockbo: Accessing hidden method Ljava/lang/Thread;->getUncaughtExceptionPreHandler()Ljava/lang/Thread$UncaughtExceptionHandler; (light greylist, reflection)
    
    --------- beginning of crash
2018-12-21 11:48:11.593 6939-6973/mozilla.lockbox E/AndroidRuntime: FATAL EXCEPTION: DefaultDispatcher-worker-2
    Process: mozilla.lockbox, PID: 6939
    kotlinx.coroutines.CompletionHandlerException: Exception in completion handler InvokeOnCompletion[InvokeOnCompletion@9ec00af] for DeferredCoroutine{Cancelled}@80bccbc
        at kotlinx.coroutines.JobSupport.notifyCompletion(JobSupport.kt:1355)
        at kotlinx.coroutines.JobSupport.completeStateFinalization(JobSupport.kt:292)
        at kotlinx.coroutines.JobSupport.tryFinalizeFinishingState(JobSupport.kt:225)
        at kotlinx.coroutines.JobSupport.tryMakeCompleting(JobSupport.kt:792)
        at kotlinx.coroutines.JobSupport.makeCompletingOnce$kotlinx_coroutines_core(JobSupport.kt:735)
        at kotlinx.coroutines.AbstractCoroutine.resumeWith(AbstractCoroutine.kt:117)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:45)
        at kotlinx.coroutines.DispatchedTask.run(Dispatched.kt:236)
        at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:594)
        at kotlinx.coroutines.scheduling.CoroutineScheduler.access$runSafely(CoroutineScheduler.kt:60)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:742)
     Caused by: org.mozilla.fxaclient.internal.FxaException$Unspecified: Network error: https://profile.accounts.firefox.com/v1/profile: an error occurred trying to connect: failed to lookup address information: No address associated with hostname
        at org.mozilla.fxaclient.internal.FxaException$Companion.fromConsuming$fxaclient_release(FxaException.kt:25)
        at org.mozilla.fxaclient.internal.FirefoxAccount.getProfile(FirefoxAccount.kt:197)
        at mozilla.components.service.fxa.FirefoxAccount$getProfile$1.invokeSuspend(FirefoxAccount.kt:61)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:32)
        at kotlinx.coroutines.DispatchedTask.run(Dispatched.kt:236) 
        at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:594) 
        at kotlinx.coroutines.scheduling.CoroutineScheduler.access$runSafely(CoroutineScheduler.kt:60) 
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:742) 
2018-12-21 11:48:11.599 6939-6973/mozilla.lockbox D/io.sentry.android.event.helper.AndroidEventBuilderHelper: Proguard UUIDs file not found.
2018-12-21 11:48:11.602 6939-6939/mozilla.lockbox D/OpenGLRenderer: Skia GL Pipeline
2018-12-21 11:48:11.640 6939-6991/mozilla.lockbox I/Adreno: QUALCOMM build                   : 3f88ca2, I42f6fe38fb
    Build Date                       : 07/13/18
    OpenGL ES Shader Compiler Version: EV031.24.00.00
    Local Branch                     : 50.04
    Remote Branch                    : 
    Remote Branch                    : 
    Reconstruct Branch               : 
2018-12-21 11:48:11.640 6939-6991/mozilla.lockbox I/Adreno: Build Config                     : S P 4.0.10 AArch64
```

**Comment by sashei** [comment]
@linuxwolf is that before or after the most recent commit? I think I may have resolved the `getProfile` one.

**Comment by linuxwolf** [comment]
> @linuxwolf is that before or after the most recent commit? I think I may have resolved the `getProfile` one.

after

**Comment by sashei** [comment]
repro'ing

**Pull request merged**